src/WebJobs.Extensions.DurableTask/ActorScheduler/SchedulerState.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+using DurableTask.Core;
+using DurableTask.Core.Serializing;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    /// The persisted state of an actor scheduler, as handed forward between ContinueAsNew instances.
+    /// </summary>
+    [JsonConverter(typeof(CustomJsonConverter<SchedulerState>))]
+    internal class SchedulerState : CustomJsonConverter<SchedulerState>.IConversion
+    {
+        /// <summary>
+        /// Whether this actor exists or not.
+        /// </summary>
+        public bool ActorExists { get; set; }
+
+        /// <summary>
+        /// The serialized actor state. This can be stale while CurrentStateView != null.
+        /// </summary>
+        public string ActorState { get; set; }
+
+        /// <summary>
+        /// The queue of waiting operations, or null if none.
+        /// </summary>
+        public Queue<OperationMessage> Queue { get; set; }
```

Comment:
Does the `Queue` property ever get assigned from within another class? If not, I suggest we change `set` into `private set`.